gradle/tests.gradle

```diff
@@ -115,6 +115,15 @@ if (hasTestsOfCategory(project, "DynamoDb")) {
     }
 }
 
+if (hasTestsOfCategory(project, "Ehcache3")) {
```

Comment:
Reminder: if we use an ehcache tag for all ehcache work in CAS, this would need to be updated as well.